File: 125-valid-palindrome/valid-palindrome.java
class Solution {
    public boolean isPalindrome(String s) {
        String rmNonAlphaNum = s.replaceAll("[^a-zA-Z0-9]", "").toLowerCase();
        String reverseString = new StringBuilder(rmNonAlphaNum).reverse().toString();
        return rmNonAlphaNum.equals(reverseString);
    }
}

File: 125-valid-palindrome/valid-palindrome.js
/**
 * @param {string} s
 * @return {boolean}
 */
var isPalindrome = function(s) {

    let lowerStr = s.toLowerCase().replace(/[^a-z0-9]/g,"");
    let revStr = lowerStr.split("").reverse().join("");
    return lowerStr === revStr;
};